### DISCUSSION TITLE
Add Community Health Files

### DISCUSSION THREAD

**Discussion by vedxp** [libre-tube/LibreTube#1627] [General]
### Describe your suggested feature

- Enforce Code Of Conduct to the community.

>Here's a quicklink: [Add COC](https://github.com/libre-tube/LibreTube/community/code-of-conduct/new?filename=CODE_OF_CONDUCT.md) 

- Add Contributing Guidelines

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Would be great if someone could make a PR for it, I don't really have time to read into such topics ;)

**Comment by vedxp** [comment]
> Would be great if someone could make a PR for it, I don't really have time to read into such topics ;)

Convert it to discussion :)
